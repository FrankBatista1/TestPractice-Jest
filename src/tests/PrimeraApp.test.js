import { shallow } from "enzyme";
import PrimeraApp from "../PrimeraApp";

describe("PrimeraApp test", () => {
  test("debe mostrar <PrimeraApp/> correctamente", () => {
    const saludo = "Hola, soy goku";
    const wrapper = shallow(<PrimeraApp saludo={saludo} />);

    expect(wrapper).toMatchSnapshot();
  });
  test("debe tener un subtitulo enviado por props", () => {
    const saludo = "Hola, soy Goku";
    const subtitulo = 'Soy un subtitulo'

    const wrapper = shallow(<PrimeraApp saludo={saludo} subtitulo={subtitulo} />);

    const textoParrafo = wrapper.find('p').text().trim();

    expect(textoParrafo).toBe(subtitulo)
  });
});
