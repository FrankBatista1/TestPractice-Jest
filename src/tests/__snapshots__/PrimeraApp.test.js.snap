// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrimeraApp test debe mostrar <PrimeraApp/> correctamente 1`] = `
<Fragment>
  <h1>
     
    Hola, soy goku
     
  </h1>
  <p>
    Soy un subtitulo
  </p>
</Fragment>
`;
